modules/secured-cluster-configuration-options-operator.adoc

```diff
@@ -164,8 +164,7 @@ Use Scanner configuration settings to modify the local cluster scanner for the i
 | Use this parameter to override the default resource requests for Scanner V4 DB.
 
 | `scannerV4.db.persistence.persistentVolumeClaim.claimName`
-| The name of the PVC to manage persistent data for Scanner V4.
-If no PVC with the given name exists, it is created. The default value is `scanner-v4-db` if not set. To prevent data loss, the PVC is not removed automatically when Central is deleted.
+| The name of the PVC to manage persistent data for Scanner V4. You can use a PVC, which is the recommended default if a default storage class exists on the cluster. You could also use a host path or an `emptyDir` volume if Scanner V4 is running on a secured cluster without a default storage class. If you did not configure a default storage class, ephemeral storage is used.
```

Comment:
I'd remove sentence "You could also use a host path or an `emptyDir` volume if Scanner V4 is running on a secured cluster without a default storage class."
'host path' is not supported and emptyDir is ephemeral storage.
Next sentence I'd rephrase to:
If no default storage class exist and it is not specifically set in persistentVolumeClaim.storageClassName parameter, ephemeral storage will be used.

Below in scannerV4.db.persistence.persistentVolumeClaim.storageClassName section - last sentence needs to be rephrased to:
If your cluster is not configured with a default storage class, and the value is not provided for this parameter - ephemeral storage will be used.